crm_claim_rma/tests/test_picking_creation.py

```diff
@@ -39,41 +41,44 @@ def setUp(self):
         self.customer_location_id = self.env.ref(
             'stock.stock_location_customers')
 
+        sale_order_agrolait_demo = self.env.ref('sale.sale_order_1')
+        self.assertTrue(sale_order_agrolait_demo.invoice_ids,
+                        "The Order Sale of Agrolait not have Invoice")
+        invoice_agrolait = sale_order_agrolait_demo.invoice_ids[0]
+        invoice_agrolait.\
+            signal_workflow('invoice_open')
+
         # Create the claim with a claim line
-        self.claim_id = Claim.create(
+        self.claim_id = claim.create(
             {
                 'name': 'TEST CLAIM',
-                'number': 'TEST CLAIM',
-                'claim_type': 'customer',
+                'code': '/',
+                'claim_type': self.env.ref('crm_claim_type.'
+                                           'crm_claim_type_customer').id,
                 'delivery_address_id': self.partner_id.id,
+                'partner_id': self.env.ref('base.res_partner_2').id,
+                'invoice_id': invoice_agrolait.id,
             })
-
+        self.claim_id.with_context({'create_lines': True}).\
+            _onchange_invoice_warehouse_type_date()
         self.warehouse_id = self.claim_id.warehouse_id
-        self.claim_line_id = ClaimLine.create(
-            {
-                'name': 'TEST CLAIM LINE',
-                'claim_origine': 'none',
-                'product_id': self.product_id.id,
-                'claim_id': self.claim_id.id,
-                'location_dest_id': self.warehouse_id.lot_stock_id.id,
-            })
 
     def test_00_new_product_return(self):
         """Test wizard creates a correct picking for product return
 
         """
-        wizard = self.WizardMakePicking.with_context({
+        wizard = self.wizardmakepicking.with_context({
```

Comment:
Recommended way of internal variables names is snake_case: `wizard_make_picking`, but it's not the most important here